server/builder.js

```diff
@@ -131,15 +140,17 @@ async function analyzeBundle(push) {
 	} else {
 		// support for webpack < 4
 		newChunkGroups = [
-			{ sha, chunk: 'build', sibling: 'manifest' },
-			{ sha, chunk: 'build', sibling: 'vendor' },
+			{ sha: newStat.sha, chunk: 'build', sibling: 'manifest' },
+			{ sha: newStat.sha, chunk: 'build', sibling: 'vendor' },
```

Comment:
I pushed a better fix for this bug in https://github.com/jsnajdr/icfy/commit/40a4fcfe1ecdb1ed399d2381f26ad73ccd3df448

There, I use `push.sha` property.